File-Formats/DocIO/Working-with-Paragraph.md

```diff
@@ -1765,6 +1765,86 @@ By executing the above code example, it generates output Word document as follow
 
 ![Output of Word document with Image caption](WorkingWithImages_images/ImageCaption.png)
 
+### Add SVG image
+
+To add an SVG image to a paragraph in a Word document using Syncfusion DocIO, you can use the [AppendPicture](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.IWParagraph.html#Syncfusion_DocIO_DLS_IWParagraph_AppendPicture_System_Byte___System_Byte___) API.
+
+N> To preserve the SVG image in the Word document, you need to pass both the SVG image data and the equivalent bitmap image bytes to DocIO.
+
+The following code example shows how to add SVG image in Word document.
```

Comment:
The following code example shows how to add an SVG image in a Word document.

Reply:
modified